src/main/java/org/apache/commons/lang3/ClassUtils.java

```diff
@@ -457,6 +457,44 @@ public static String getAbbreviatedName(final String className, final int len) {
     }
 
     /**
+     * <p>Gets a combination of {@link #getAllSuperclasses}(Class)} and
+     * {@link #getAllInterfaces}(Class)}, one from superclasses, one
+     * from interfaces, and so on in a breadth first way.</p>
+     *
+     * @param cls  the class to look up, may be {@code null}
+     * @return the {@code List} of superclasses in order going up from this one
```

Comment:
This should also mention Interfaces.

Can you add a since tag to this method?

Thanks!